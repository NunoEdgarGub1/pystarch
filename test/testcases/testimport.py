imported = {}
from misc import z as my_integer, c as my_other_integer
import email.mime.text as mime_text
import subprocess
