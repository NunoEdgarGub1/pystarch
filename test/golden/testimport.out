imported Dict(Unknown,Unknown) {}
mime_text Instance(object)
my_integer Num
my_other_integer Num
